spec/features/finance/schedules_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "Finance users payment breakdowns", type: :feature do
+  fscenario "viewing a schedule" do
+    given_i_am_logged_in_as_a_finance_user
+    and_there_is_a_schedule
+    when_i_click("Schedules")
+    then_i_see("Schedules")
+    and_see_table_with_schedule(@schedule)
+
+    when_i_click(@schedule.schedule_identifier)
+    then_i_see("Milestones")
+    and_see_table_of_milestones_for_schedule(@schedule)
+  end
+
+  def and_there_is_a_schedule
+    @schedule = create(:ecf_schedule)
```

Comment:
Could we follow the usual RSpec way of defining test data: `let!(schedule)` at the top of the file.

Also avoids jumping up and down the file, losing context, and it also keeps consistency with the rest of the RSpec tests (I'm aware that some other feature have been using this "cucumber" steps way of doing but there really isn't any benefit to it, also this instance variable can be modified midway through the test and you'd never know it was.